SPECS/coredns/coredns.spec

```diff
@@ -34,6 +34,7 @@ Patch0:         makefile-buildoption-commitnb.patch
 Patch1:         CVE-2023-44487.patch
 Patch2:         CVE-2023-49295.patch
 Patch3:         CVE-2024-22189.patch
+Patch4:         CVE-2024-0874.patch
```

Comment:
Seems to be a medium and not close to SLA. Seems like there's no need for fast-track, please use `main`.